### DISCUSSION TITLE
tutorial-extras/MPU6050

### DISCUSSION THREAD

**Discussion by giscus** [Air-duino/document#28] [comment]
# tutorial-extras/MPU6050

简介 本章介绍使用Air001开发板驱动MPU6050。 MPU6050使用I²C通信接口，内部整合了三轴MEMS陀螺仪、三轴MEMS加速度计和一个内置温度传感器，可以读取三轴角度，三轴加速度以及当前温度。 硬件准备 按☁️ Air001开发板入门 (/tutorial-advanced/Air001_start.html)，将Air001和`DAPL...

https://arduino.luatos.com/tutorial-extras/MPU6050.html

**Comment by Aivin40** [comment]
用软件串口的时候直接不运行程序了是为什么
